symplyphysics/laws/condensed_matter/diffusion_coefficient.py

```diff
@@ -0,0 +1,49 @@
+from sympy import (Eq, solve, exp)
```

Comment:
There are various ways to calculate diffusion coefficient. Add `from_something` to the law name.